alignak/action.py

```diff
@@ -79,8 +83,13 @@
                    '|', '{', '}', ';', '<', '>', '?', '`')
 
 
-# Try to read a fd in a non blocking mode
 def no_block_read(output):
+    """Try to read a fd in a non blocking mode
```

Comment:
Need to not use abbreviation, what is 'fd'? 
Need to use the complete word else it's not easy to understand